core/src/main/resources/error/error-classes.json

```diff
@@ -235,6 +235,11 @@
     ],
     "sqlState" : "22018"
   },
+  "CORRUPTED_TABLE_PROPERTY" : {
+    "message" : [
+      "Cannot read table property <key> as it's corrupted.<details>"
```

Comment:
Could you add sub-classes instead of `<details>`, please.

Reply:
It might be renamed to `INSUFFICIENT_TABLE_PROPERTY` with sub classes like this:
```json
"INSUFFICIENT_TABLE_PROPERTY": {
  "message" : [
    "Can't find table property:"
  ],
  "subClass" : {
    "TABLE_PROPERTY_MISSING" : {
      "message" : [
        "<key>."
      ]
    },
    "TABLE_PROPERTY_PART_MISSING": {
      "message" : [
        "<key>, <totalAmountOfParts> parts are expected..."
      ]
    }
  }
}
```

Error message examples:
- ```[INSUFFICIENT_TABLE_PROPERTY.TABLE_PROPERTY_MISSING] Can't find table property: `spark`.`sql`.`sources`.`schema`.```
- ```[INSUFFICIENT_TABLE_PROPERTY.TABLE_PROPERTY_PART_MISSING] Can't find table property: `spark`.`sql`.`sources`.`schema`.`part`.`1`, 3 parts are expected...```

What do you think? Is it fine?

Reply:
SGTM

Reply:
Done